bundle.yaml

```diff
@@ -1,23 +1,23 @@
 series: xenial
 applications: 
   postgresql: 
-    charm: "cs:postgresql-105"
+    charm: "cs:postgresql"
     num_units: 1
     annotations: 
       "gui-x": 1407
       "gui-y": 295.0000305175781
     to: 
       - "0"
   "serial-vault": 
-    charm: "cs:~jamesj/serial-vault-6"
```

Comment:
The number is the version. So this was pinned to the older version of the serial-vault charm. This change is unpinning it so that it gets the latest one.